sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -408,11 +413,26 @@ def analyze_sentiment(  # type: ignore
         docs = _validate_batch_input(documents, "language", language)
         model_version = kwargs.pop("model_version", None)
         show_stats = kwargs.pop("show_stats", False)
+        show_aspects = kwargs.pop("show_aspects", None)
```

Comment:
I think this should default to False

Reply:
I went with `None` to get the service default, @johanste is the policy to default to `None` in this case, or `False` to keep it static

Reply:
If we need to distinguish between "the application explicitly passed in this value" vs. "the application didn't provide a value, so we'll pick an appropriate default for them" for positional arguments, then we use a sentinel value as the default value. This is often `None` when `None` is not a valid value. 

Since we are dealing with `kwargs` here, it would be easier/clearer to check `if 'show_aspects' in kwargs:` - no need for sentinel values since `kwargs` inherently let's you determine if the user passed the parameter or not.

Reply:
@johanste I think I I do need to still pop kwargs in this case since the name of the parameter has changed (we have it as `show_aspects`, the service has it as `opinion_mining`). I think I'll stick with the sentinel value of `None`, holler if this is wrong